tutorial/decorators.md

```diff
@@ -0,0 +1,243 @@
+# Decorators
+
+Because Pyccel is converting a dynamically typed language (Python) to statically types ones, It imposes some *restrictions* on the code. One of them is using decorators that have certain options in order to specify how the functions or the variables should be treated in the conversion, Here are the available decorators.
```

Comment:
I would not say that decorators are restrictions, rather they are optional markers which allow the user to benefit from adv advantages that are not available in python

Reply:
True.